app/partials/errorDetail.html

```diff
@@ -1,9 +1,12 @@
 <pagination>
 
 <div class="detail-header content-l">
-    <div class="content-l_col content-l_col-2-3">
+    <div class="content-l_col content-l_col-1-3">
         <pagination-size></pagination-size>
     </div>
+    <div class="content-l_col content-l_col-1-3">
+        <code-descriptions></code-descriptions>
+    </div>
```

Comment:
Well now that you say that it seems like a good idea. Sounds like a question for a designer! @vizui what do you think?

Here's what happens right now: 
![image](https://cloud.githubusercontent.com/assets/695452/7262126/26836df2-e845-11e4-870d-6f68ef06a411.png)